src/cli/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ set(COMMON_SOURCES
     cli_dns.cpp
     cli_history.cpp
     cli_joiner.cpp
+    cli_link_metrics.cpp
```

Comment:
Should we also update https://github.com/openthread/openthread/blob/main/src/cli/BUILD.gn ?

Reply:
Updated.